README.md

```diff
@@ -84,11 +97,23 @@ fetch(url_to_geotiff_file)
 });
 ```
 
-# Advanced Capabilities
+<!-- ## Options -->
+<!-- todo: add a table of options for GeoRasterLayer -->
+
+### Methods
+
+| Method                                    | Returns | Description                                                                                                                                                                                                                     |
+| ----------------------------------------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| updateColors(pixelValuesToColorFn, options) | this    | Causes the tiles to redraw without clearing them first. It uses the updated `pixelValuesToColorFn` function. You can leave the console debugs for this method on by setting `debugLevel = 1` in the options (It defaults to 0). |
```

Comment:
Could we re-word a little?
```
Causes the tiles to redraw without clearing them first. It uses the updated `pixelValuesToColorFn` function. You can set a debugLevel specific to this function by passing in an options object with a debugLevel property.  For example, you can turn on the console debugs for this method by setting `debugLevel = 1` in the options (even if you created the layer with `debugLevel = 0`).
```

Reply:
I'd like to default debugLevel to whatever this.debugLevel is rather than zero.

Reply:
This is an elegant solution. I'll get it done now. 

Can I suggest we default the global `debugLevel` to 0 in production? I've added https://github.com/GeoTIFF/georaster-layer-for-leaflet/issues/57 to handle this which I'm happy to do when I'm back in the new year.